cdk/lib/personalised-fronts.ts

```diff
@@ -0,0 +1,124 @@
+import { GuParameter, type GuStack } from '@guardian/cdk/lib/constructs/core';
+import { aws_sns, Duration } from 'aws-cdk-lib';
+import {
+	Alarm,
+	ComparisonOperator,
+	Metric,
+	TreatMissingData,
+} from 'aws-cdk-lib/aws-cloudwatch';
+import { SnsAction } from 'aws-cdk-lib/aws-cloudwatch-actions';
+import {
+	AccountPrincipal,
+	Effect,
+	PolicyDocument,
+	PolicyStatement,
+	Role,
+} from 'aws-cdk-lib/aws-iam';
+import type { IBucket } from 'aws-cdk-lib/aws-s3';
+import { CfnOutput } from 'aws-cdk-lib/core';
+import { Construct } from 'constructs';
+import type { ExternalParameters } from './external_parameters';
+
+interface PersonalisedFrontsProps {
+	destBucket: IBucket;
+	externalParameters: ExternalParameters;
+}
+
+export class PersonalisedFronts extends Construct {
+	constructor(scope: GuStack, name: string, props: PersonalisedFrontsProps) {
+		super(scope, name);
+
+		const base_path = 'personalised/curation';
+
+		const dataTechCrossAccountARN = new GuParameter(
+			scope,
+			'PersonalisedFrontsCrossAccountARN',
+			{
+				fromSSM: true,
+				default: `/INFRA/recipes-backend/personalised-fronts-fetcher/data-tech-account-arn`,
+				type: 'String',
+			},
+		);
+
+		const dataTechCrossAccountID = new GuParameter(
+			scope,
+			'PersonalisedFrontsSrcAcct',
+			{
+				fromSSM: true,
+				default: `/INFRA/recipes-backend/personalised-fronts-fetcher/data-tech-account-id`,
+				type: 'String',
+			},
+		);
+
+		const iamRole = new Role(this, 'FetcherRole', {
+			roleName: `personalised-fronts-fetcher-${scope.stage}`,
+			assumedBy: new AccountPrincipal(dataTechCrossAccountID.valueAsString),
+			inlinePolicies: {
+				S3Put: new PolicyDocument({
+					statements: [
+						new PolicyStatement({
+							effect: Effect.ALLOW,
+							actions: ['s3:PutObject', 's3:GetObject'],
```

Comment:
Do you need to put objects from that lambda? My understanding was that it was a read-only lambda